src/components/Footer/Footer.tsx

```diff
@@ -1,15 +1,16 @@
 import styled from 'styled-components';
 import { useTranslation } from 'react-i18next';
 
+import { breakpoints, palette } from '../variables';
 import FooterLogo from '@/static/img/footer-logo.svg';
-import { palette, basicSourceSansText, breakpoints } from '../variables';
+import Text from '../Text';
 
 const Footer = () => {
-  const { t } = useTranslation();
+  const { t } = useTranslation('common');
 
   return (
     <StyledFooter>
-      <StyledFooterText>{t('footer')}</StyledFooterText>
+      <StyledFooterText variant="footer">{t('footer')}</StyledFooterText>
```

Comment:
This change was a nice one, maybe we can just rename it to FooterText. 
I dont mind having the Styled-prefix, but I would only use it if the original namespace is taken

Reply:
Will do! 👍